src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,3 @@
+$sec: 10s;
+$min: 10min;
+$hands: 'seconds' $sec, 'minutes' $min;
```

Comment:
The `$hands` variable is intended to be a list or map for use in an `@each` loop, but the syntax is incorrect. You should define it as a list or map. For example, use a list: `('seconds' $sec, 'minutes' $min)` or a map: `('seconds': $sec, 'minutes': $min)`.